form_jsons/config/development.json

```diff
@@ -5,5 +5,6 @@
     "env": "development",
     "safelist": ["funding-service-design-application-store-dev.london.cloudapps.digital", "funding-service-design-application-store-test.london.cloudapps.digital"],
     "jwtAuthCookieName": "fsd_user_token",
-    "jwtAuthenticationUrl": "https://authenticator.dev.fundingservice.co.uk/service/magic-links/invalid"
+    "jwtAuthenticationUrl": "https://authenticator.dev.fundingservice.co.uk/service/magic-links/invalid",
```

Comment:
This url will need to be updated obviously when we have the /signout (signed out) route available on the authenticator